src/Backups/BackupIO_S3.h

```diff
@@ -39,8 +39,8 @@ class BackupReaderS3 : public IBackupReader
 private:
     S3::URI s3_uri;
     std::shared_ptr<Aws::S3::S3Client> client;
-    UInt64 max_single_read_retries;
     ReadSettings read_settings;
+    S3Settings::ReadWriteSettings rw_settings;
```

Comment:
may be rename this to something like `s3_settings` or `s3_rw_settings`? because `rw_settings` and `read_settings` are now very similar and judging by their names it is not obvious what is the difference unless you take a look at the types.

Reply:
hm maybe `S3Settings::RequestSettings`? (as it might be used not only for put and get, for head also applicable)

> Also should we make it in this PR?

I think it does not matter, whatever is easier for you